Documentation/contributing/testing/ci.rst

```diff
@@ -221,6 +224,16 @@ For a full list of Jenkins Jobs, see `Jenkins
 <https://jenkins.cilium.io/view/PR/>`_. Trigger phrases are configured within
 each job's build triggers advanced options.
 
+``test-older-k8s`` allows testing of all K8s versions 1.14 through 1.19 (it will trigger the appropriate ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs).
```

Comment:
This `1.14 through 1.19` will be hard to keep up-to-date :/. Could we rephrase we something like “all supported versions older than the latest” or similar?

Reply:
I actually had that in the first version, but there's also `net-next` (1.13) which is not ran, so technically it is "all supported versions older than the latest and net-next", and I found that a bit strange.

If ease of maintenance is more important than accuracy, or I misunderstood and `net-next` is something else entirely and we don't support 1.13 anyway, then I'll flip it back. Otherwise maybe we need another idea.

Reply:
Accuracy is more important I guess, but it's no longer accurate once you forget to update :/. What about having the numbers as an example, something like:
> all supported versions except the latest and net-next (Example: As of this writing we have K8s 1.13 running with net-next, and K8s 1.20 the latest version. So the phrase will trigger runs for _all K8s versions from 1.14 to 1.19_)

This way it's explicit, but also more future-proof?

Reply:
I've got an idea:

> ``test-older-k8s`` allows testing all non-required K8s jobs (it will trigger the appropriate ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs).

Reply:
I've settled on a mix:

> ``test-older-k8s`` allows testing older supported K8s versions, i.e. all
non-required K8s jobs, by triggering all appropriate
``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs.
For example, at the time of writing, it is identical to manually running
``test-X.XX-4.9`` for versions 1.14 through 1.19.
If a specific K8s version fails, it can be re-ran using ``retest-X.XX-4.9``
(with ``X.XX`` being the specific K8s version).